mri/tests/test_sensitivity_extraction.py

```diff
@@ -63,23 +63,26 @@ def test_extract_k_space_center_3D(self):
             data_thresholded)
 
     def test_extract_k_space_center_2D(self):
-        """ Ensure that the extracted k-space center is right"""
-        _mask = np.ones((self.N, self.N))
-        _samples = convert_mask_to_locations(_mask)
+        """ Ensure that the extracted k-space center is right,
+        send density compensation also and vet the code path"""
+        mask = np.ones((self.N, self.N))
+        samples = convert_mask_to_locations(mask)
         Img = (np.random.randn(self.num_channel, self.N, self.N) +
                1j * np.random.randn(self.num_channel, self.N, self.N))
         Nby2_percent = self.N * self.percent / 2
         low = int(self.N / 2 - Nby2_percent)
         high = int(self.N / 2 + Nby2_percent + 1)
         center_Img = Img[:, low:high, low:high]
         thresh = self.percent * 0.5
-        data_thresholded, samples_thresholded = \
+        data_thresholded, samples_thresholded, dc = \
             extract_k_space_center_and_locations(
                 data_values=np.reshape(Img, (self.num_channel,
                                              self.N * self.N)),
-                samples_locations=_samples,
+                samples_locations=samples,
                 thr=(thresh, thresh),
-                img_shape=(self.N, self.N))
+                img_shape=(self.N, self.N),
+                density_comp=np.ones(samples.shape[0])
```

Comment:
same here (i.e. with and without dc)

Reply:
Well here it makes sense, as we arent testing the other code paths at all. Adding this.

Reply:
you didn't do it right? or am I missing sthg?

Reply:
I did add it, it is in the same test. Below you can see that I added another Smap estimation